cmd/rqlite/main.go

```diff
@@ -23,6 +24,7 @@ type argT struct {
 	Prefix      string `cli:"P,prefix" usage:"rqlited HTTP URL prefix" dft:"/"`
 	Insecure    bool   `cli:"i,insecure" usage:"do not verify rqlited HTTPS certificate" dft:"false"`
 	Credentials string `cli:"u,user" usage:"set basic auth credentials in form username:password"`
+	CACert      string `cli:"ca-cert" usage:"path to trusted X.509 root CA certificate"`
```

Comment:
Just so the style remains consistent, can you also add a single letter option i.e. `cli:c,ca-cert"`.

Reply:
Yes, added.